components/feature/top-sites/src/main/java/mozilla/components/feature/top/sites/PinnedSiteStorage.kt

```diff
@@ -88,6 +88,20 @@ class PinnedSiteStorage(context: Context) {
         pinnedSiteDao.updatePinnedSite(pinnedSite)
     }
 
+    /**
+     * Edits the given pinned site.
+     *
+     * @param site The pinned site.
+     * @param title The new title for the top site.
+     * @param url The new url for the top site.
+     */
+    suspend fun editPinnedSite(site: TopSite, title: String, url: String) = withContext(IO) {
```

Comment:
Can we also make sure there is sufficient unit tests for this in PinnedSiteStorageTest.

Reply:
Yes we can, thanks, I missed that!